resources/kcp/charts/keda/README.md

```diff
@@ -0,0 +1,147 @@
+| `resources`                                                | Manage resource request & limits of all KEDA workloads ([docs](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)) | `{}` |
+| `resources.operator`                                       | Manage resource request & limits of KEDA operator pod ([docs](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)) | `` |
+| `resources.metricServer`                                   | Manage resource request & limits of KEDA metrics apiserver pod ([docs](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)) | `` |
+| `nodeSelector`                                             | Node selector for pod scheduling ([docs](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/)) | `{}` |
+| `tolerations`                                              | Tolerations for pod scheduling ([docs](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)) | `{}` |
+| `affinity`                                                 | Affinity for pod scheduling ([docs](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/)) | `{}` |
+| `priorityClassName`                                        | Pod priority for KEDA Operator and Metrics Adapter ([docs](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/)) | `` |
+| `env`                                                      | Additional environment variables that will be passed onto KEDA operator and metrics api service | `` |
+| `http.timeout` | The default HTTP timeout to use for all scalers that use raw HTTP clients (some scalers use SDKs to access target services. These have built-in HTTP clients, and the timeout does not necessarily apply to them) | `` |
+| `service.annotations`                                      | Annotations to add the KEDA Metric Server service | `{}`                                        |
+| `service.portHttp`                                         | Service HTTP port for KEDA Metric Server service | `80`                                        |
+| `service.portHttpTarget`                                   | Service HTTP port for KEDA Metric Server container | `8080`                                        |
+| `service.portHttps`                                        | HTTPS port for KEDA Metric Server service | `443`                                        |
+| `service.portHttpsTarget`                                  | HTTPS port for KEDA Metric Server container | `6443`                                        |
+| `prometheus.metricServer.enabled`                          | Enable metric server prometheus metrics expose | `false`
+| `prometheus.metricServer.port`                             | HTTP port used for exposing metrics server prometheus metrics | `9022`

```

Comment:
```suggestion
| `prometheus.metricServer.port`                             | HTTP port to expose metrics server Prometheus metrics | `9022`

```

Reply:
See first comment. This was resolved after our Slack discussion.